apps/lib/s_cb.c

```diff
@@ -770,8 +770,9 @@ static STRINT_PAIR signature_tls13_scheme_list[] = {
     {"rsa_pss_pss_sha384",     0x080a /* TLSEXT_SIGALG_rsa_pss_pss_sha384 */},
     {"rsa_pss_pss_sha512",     0x080b /* TLSEXT_SIGALG_rsa_pss_pss_sha512 */},
     {"gostr34102001",          0xeded /* TLSEXT_SIGALG_gostr34102001_gostr3411 */},
-    {"gostr34102012_256",      0xeeee /* TLSEXT_SIGALG_gostr34102012_256_gostr34112012_256 */},
-    {"gostr34102012_512",      0xefef /* TLSEXT_SIGALG_gostr34102012_512_gostr34112012_512 */},
+    /* rfc9189 */
+    {"gostr34102012_256",      0x0840 /* TLSEXT_SIGALG_gostr34102012_256_gostr34112012_256 */},
```

Comment:
@mattcaswell - is this likely to be an interop problem?

Reply:
I don't know if @beldmit should be in the loop as well?

Reply:
We could keep the existing values as well.